docs/src/estimators_cheatsheet.md

```diff
@@ -0,0 +1,298 @@
+```
+
+And its gradient is given by:
+
+```math
+\begin{aligned}
+\phi_{AIE, }(W, T, Y) &= \phi_{CM_{t_{1,case}, t_{2,case}}}(W, T, Y) - \phi_{CM_{t_{1,case}, t_{2,control}}}(W, T, Y) - \phi_{CM_{t_{1,control}, t_{2,case}}}(W, T, Y) + \phi_{CM_{t_{1,control}, t_{2,control}}}(W, T, Y) \\
+&=  H(W, T)(Y − Q_Y(W, T)) + C(W) − ATE_{t_{case}, t_{control}}(\mathbb{P})
+\end{aligned}
+```
+
+with:
+
+```math
+\begin{aligned}
+  \begin{cases}
+    H(W, T) &= \frac{(-\mathbb{1}(T_1 \in (t_{1, case}, t_{1, control})))^{T_1=t_{1, control}}(-\mathbb{1}(T_2 \in (t_{2, case}, t_{2, control})))^{T_2=t_{2, control}}}{g(W, T)} \\
+    C(W) &= (Q_Y(W, t_{1, case}, t_{2, case}) - Q_Y(W, t_{1, case}, t_{2, control}) - Q_Y(W, t_{1, control}, t_{2, case}) + Q_Y(W, t_{1, control}, t_{2, control}))
+  \end{cases}
+\end{aligned}
+```
+
+For higher-order interactions the two factors ``H`` and ``C`` can be similarly inferred.
+
+## Asymptotic Analysis of Plugin Estimators
+
+The theory is based on the von Mises expansion (functional equivalent of Taylor's expansion) which reduces due to the fact that the gradient satisfies: ``\mathbb{E}_{\mathbb{P}}[\phi_{\mathbb{P}}(Z)] = 0``.
+
+```math
+\begin{aligned}
+\Psi(\hat{\mathbb{P}}) − \Psi(\mathbb{P}) &= \int \phi_{\hat{\mathbb{P}}}(z) \mathrm{d}(\hat{\mathbb{P}} − \mathbb{P})(z) + R_2(\hat{\mathbb{P}}, \mathbb{P}) \\
+&= - \int \phi_{\hat{\mathbb{P}}}(z) \mathrm{d}\mathbb{P}(z) + R_2(\hat{\mathbb{P}}, \mathbb{P})
+\end{aligned}
+```
+
+This suggests that a plugin estimator, one that simply evaluates ``\Psi`` at an estimator ``\hat{\mathbb{P}}`` of ``\mathbb{P}``, is biased. This bias can be elegantly decomposed in four terms by reworking the previous expression:
+
+```math
+\Psi(\hat{\mathbb{P}}) − \Psi(\mathbb{P}) = \mathbb{P}_n\phi_{\mathbb{P}}(Z) - \mathbb{P}_n\phi_{\mathbb{\hat{P}}}(Z) + (\mathbb{P}_n − \mathbb{P})(ϕ_{\mathbb{\hat{P}}}(Z) − \phi_{\mathbb{P}}(Z)) + R_2(\mathbb{\hat{P}}, \mathbb{P})
+```
+
+### 1. The Asymptotically Linear Term
+
+```math
+\mathbb{P}_n\phi_{\mathbb{P}}(Z)
+```
+
+By the central limit theorem, it is asymptotically normal with variance ``Var[\phi]/n``, it is used to build confidence interval.
+
+### 2. The Bias Term
+
+```math
+- \mathbb{P}_n\phi_{\mathbb{\hat{P}}}(Z)
+```
+
+This is the term both the One-Step estimator and the Targeted Maximum-Likelihood estimator deal with.
+
+### 3. The Empirical Process Term
+
+```math
+(\mathbb{P}_n − \mathbb{P})(ϕ_{\mathbb{\hat{P}}}(Z) − \phi_{\mathbb{P}}(Z))
+```
+
+This can be shown to be of order ``o_{\mathbb{P}}(\frac{1}{\sqrt{n}})`` if ``\phi_{\hat{\mathbb{P}}}`` converges to ``\phi_{\mathbb{P}}`` in ``L_2(\mathbb{P})`` norm, that is:
+
+```math
+\int (\phi_{\hat{\mathbb{P}}}(Z) - \phi_{\mathbb{P}}(Z) )^2 d\mathbb{P}(Z) = o_{\mathbb{P}}(\frac{1}{\sqrt{n}})
```

Comment:
Suggestion: Write " o_{\mathbb{P}} \left( \frac{1}{\sqrt{n}} \right) " so that the size of the brackets matches the size of the expression. Similarly for some expressions below.